kubernetes/src/test/resources/kubernetes/kubernetes.tests.bom

```diff
@@ -89,3 +89,12 @@ brooklyn.catalog:
             kubectl get deployments | grep workload- | wc -l
           assertOut:
             contains: '0'
+        - type: org.apache.brooklyn.test.framework.TestHttpCall
+           name: "11. Check Prometheus UI is Reachable"
+           url: >
```

Comment:
Great to see this added to the test cases!

Very minor preference (feel free to ignore, or to argue against - certainly not important for getting this PR merged):

I've been writing this kind of think like the yaml below. It feels simpler to read than the very long string.
```
         url:
           $brooklyn:formatString:
             - "http://%s:%s"
             - $brooklyn:entity("prometheus").attributeWhenReady("host.address")
             - $brooklyn:entity("prometheus").attributeWhenReady("kubernetes.service.port")
```